Hippo.Web/Controllers/AccountController.cs

```diff
@@ -142,7 +142,7 @@ public async Task<ActionResult> Reject(int id, [FromBody] RequestRejectionModel
     public async Task<ActionResult> Create([FromBody] AccountCreateModel model)
     {
         var currentUser = await _userService.GetCurrentUser();
-        model.SshKey = Regex.Replace(model.SshKey, @"(?<!ssh-rsa)\s+", "");
+        model.SshKey = Regex.Replace(model.SshKey, @"(?<!ssh-rsa)\s+(([\w\.\-]+)@([\w\-]+\.?)+)?", "");
```

Comment:
If they have junk in their SSH, wouldn't it be better to validate that instead of replace it?

Reply:
It's optional metadata. Removing the whitespace ended up concatenating it to the actual key. So this just strips the metadata out.

Reply:
you are correct anyway, but this is just temporary while I am working on a 2 for 1 solution for validation and formatting